django/http/response.py

```diff
@@ -141,6 +137,16 @@ def charset(self):
     def charset(self, value):
         self._charset = value
 
+    @property
+    def reason_phrase(self):
+        if self._reason_phrase is not None:
```

Comment:
It makes sense to me that you would leave it unset unless `self.reason_phase` explicitly sets it.